src/util.js

```diff
@@ -86,9 +86,16 @@ export function getLessons(lessonContext, readmeContext, courseContext) {
   }, {});
 }
 
-export function getCourseInfo(courseName) {
-  const courseInfo = require('onlyContent!lessonSrc/' + courseName + '/index.md');
-  return courseInfo.content;
+/**
+* Returns /course/index_(ISO_CODE) if it exists, returns /course/index if not.
+**/
+export function getCourseInfo(courseName, language) {
+  try {
+    return require('onlyContent!lessonSrc/' + courseName + '/index_' + language + '.md').content;
+  }
+  catch(err) {
+    return require('onlyContent!lessonSrc/' + courseName + '/index.md').content;
```

Comment:
I don't like that the "exception" is the most normal thing. I prefer that the exception (i.e. the catch clause) should only be used when something "exceptional" happens. Although I must admit that the solution is pretty neat, I would rather that we find a way to check if the language-files exist other than using try-catch. If this is hard to do / makes the code much more complicated / harder to read / harder to maintain, we could keep the way it is, but with a comment as to why we are allowing it here.

If the case is that we are phasing out index.md and just keeping the index with the languages, i.e. we are expecting that there should always be an index_XX.md, then I am more inclined to keep this solution...